maintainers/maintainer-list.nix

```diff
@@ -5777,6 +5777,11 @@
       fingerprint = "7573 56D7 79BB B888 773E  415E 736C CDF9 EF51 BD97";
     }];
   };
+  raboof = {
```

Comment:
Adding yourself as a maintainer is preferred to be a separate commit.
Something like, `maintainers: add raboof`. Then apply the MozPhab commit.